src/components/cards/sub-components/card-container.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { commonStyles } from '../cards-styles';
+import './cards.css';
+
+export const CardContainer = ({ children }: { children: React.ReactNode }) => {
```

Comment:
duvida: se vc tipar como React.FC, a prop children é tipada pra vc automaticamente?

```suggestion
export const CardContainer: React.FC = ({ children }) => {
```

da uma testada e me fala, pq dependendo da versao do react, nao

Reply:
me retornou essa mensagem: A propriedade 'children' não existe no tipo '{}'.

Reply:
humm ok ok, entao esquece hahah